posixlib/src/main/resources/scala-native/termios.c

```diff
@@ -37,8 +45,16 @@ int scalanative_termios_onlcr() { return ONLCR; }
 int scalanative_termios_ocrnl() { return OCRNL; }
 int scalanative_termios_onocr() { return ONOCR; }
 int scalanative_termios_onlret() { return ONLRET; }
+#ifdef OFDEL
```

Comment:
Same as in `ernno.c`, can we move conditions to the body of the functions.

Reply:
Done